tldraw/apps/tldraw-logseq/src/hooks/usePaste.ts

```diff
@@ -29,8 +29,8 @@ import { LogseqContext, LogseqContextValue } from '../lib/logseq-context'
 
 const isValidURL = (url: string) => {
   try {
-    new URL(url)
-    return true
+    const parsedUrl = new URL(url)
+    return parsedUrl.host && ['http:', 'https:'].includes(parsedUrl.protocol)
```

Comment:
new URL(url) was successful for unsupported  protocols, but also for non url strings that contained `:`